src/main/java/be/syntra/devshop/DevshopBack/entities/Cart.java

```diff
@@ -0,0 +1,64 @@
+package be.syntra.devshop.DevshopBack.entities;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+import org.hibernate.annotations.Cascade;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Cart {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "cart_id")
+    private Long id;
+
+
+    @OneToOne(targetEntity = User.class, mappedBy = "activeCart")
+    @JsonIgnore
+    private User user;
+
+    @NotNull
+    @Column(name = "cartCreationDateTime")
+    @JsonFormat
+            (shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
+
+    private LocalDateTime cartCreationDateTime;
+
+    @NotNull
+    @Column(name = "products")
+    @OneToMany
+    @Cascade(org.hibernate.annotations.CascadeType.ALL)
```

Comment:
remove @Cascade and add to @OneToMany(cascase = CascadeType.ALL). This way, we don't mix javax.persistence and Spring annotations

Reply:
please do so for all files with this annotation